spec/features/create_game_spec.rb

```diff
@@ -0,0 +1,181 @@
+require 'rails_helper'
+
+RSpec.describe 'games/create' do
+  let!(:players) do
+    create_list(:player, 6) do |player, index|
+      unique_id = index + 1
+      player.id = unique_id
+      player.name = "player#{unique_id}"
+      player.save
+    end
+  end
+
+  let!(:decks) do
+    create_list(:deck, 6) do |deck, index|
+      unique_id = index + 1
+      deck.id = unique_id
+      deck.name = "deck#{unique_id}"
+      deck.player_id = unique_id
```

Comment:
This is hacky instead create a player:
```rb
deck.player = create(:player)
```

Reply:
`create` -> creates the object in the database which generates a unique id
`build` -> builds the object in memory without an id